Tools/autotest/common.py

```diff
@@ -1375,6 +1375,17 @@ def test_arm_feature(self):
             self.mav.wait_heartbeat()
             self.set_rc(arming_switch, 1000)
             self.set_rc(interlock_channel, 1000)
+            if self.mav.mav_type is not mavutil.mavlink.MAV_TYPE_HELICOPTER:
+                self.start_test("Test motor interlock enable can't be set while disarmed")
+                self.set_rc(interlock_channel, 2000)
+                try:
+                    if self.wait_servo_channel_value(interlock_channel, 2000, timeout=20):
```

Comment:
@khancyr  thanks for including these tests in a new PR!!   there is an issue with this test though.  first, heli's don't use the interlock switch aux option.  interlock is automatically tied to channel 8.  So this will probably work with mult's that use the interlock switch aux option but not heli's.  I will have to test this to see if it works on heli's   the other issue is that if this correctly identifies channel 8 as the interlock channel, then heli's don't always set it to 2000 to enable interlock.   that is why my logic looks to see that it is not SERVO8_MIN value.

Reply:
arf yes I made a mistake when reodering... Heli is the only case were we don't want to use channel 8 and set the interlock option. I will correct that tomorrow. I test those on default heli on SITL and it is was working.
Yes, I forget to use the servo8_min, I will correct that too

Reply:
corrections done !